pkg/commands/compute/language_toolchain.go

```diff
@@ -1,10 +1,534 @@
 package compute
 
 import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
 	"os/exec"
+	"path/filepath"
+	"regexp"
 	"strings"
+	"time"
+
+	"github.com/Masterminds/semver/v3"
+	fsterr "github.com/fastly/cli/pkg/errors"
+	fstexec "github.com/fastly/cli/pkg/exec"
+	"github.com/fastly/cli/pkg/filesystem"
+	"github.com/fastly/cli/pkg/manifest"
+	"github.com/fastly/cli/pkg/text"
+	toml "github.com/pelletier/go-toml"
 )
 
+// SDKErrMessageFormat is a format string that can be used by the
```

Comment:
Here's where the majority of the new functionality (i.e the new abstraction layer) exists.